model_analyzer/cli/cli.py

```diff
@@ -121,13 +116,87 @@ def _add_arguments(self):
             type=str,
             default='perf_analyzer',
             help='The full path to the perf_analyzer binary executable')
+        self._parser.add_argument(
+            '--triton-launch-mode',
+            type=str,
+            choices=['local', 'docker', 'remote'],
+            default='local',
+            help="The method by which to launch Triton Server. "
+                 "'local' assumes tritonserver binary is available locally. "
+                 "'docker' pulls and launches a triton docker container with the specified version. "
+                 "'remote' connects to a running server using given http, grpc and metrics endpoints. "
+        )
+        self._parser.add_argument(
+            '-v',
```

Comment:
Don't we want to reserve `-v` for the verbose flag or the model analyzer version?

Reply:
Yeah this makes sense,  but both this and `--base-duration` are original memory-analyzer CLI options. I didn't want to make any breaking changes. However, `-v` should be reserved.